packages/appstore/src/components/wallet-header/wallet-header.tsx

```diff
@@ -7,31 +7,36 @@ import WalletHeaderTitle from './wallet-header-title';
 import WalletHeaderBalance from './wallet-header-balance';
 import { TAccountCategory, TAccountStatus, TWalletShortcode, TWalletCurrency } from 'Types';
 import { getWalletHeaderButtons } from 'Constants/utils';
+import { formatMoney } from '@deriv/shared';
 import './wallet-header.scss';
 
 type TWalletHeader = {
     account_type: TAccountCategory;
     shortcode?: TWalletShortcode;
     currency?: TWalletCurrency;
-    balance?: string;
+    balance?: number;
```

Comment:
@sergei-deriv why in the 'packages/appstore/src/components/wallet-header/wallet-header-balance.tsx' file the type for balance is string and here we use it as number?

Reply:
@mahdiyeh-deriv it's because I use `formatMoney(currency, balance, true)` for component WalletHeaderBalance to show balance as string, for example, number `10567.32` will be converted to `10,567.32` string

Reply:
@mahdiyeh-deriv I did this way because WalletCard has `balance` as string and I made decision to do the same way and leave component dumb